GVFS/GVFS.Common/Maintenance/LooseObjectsStep.cs

```diff
@@ -65,15 +113,18 @@ protected override void PerformMaintenance()
                         }
                     }
 
-                    int beforeCount = this.CountLooseObjects();
+                    int beforeLooseObjectsCount = this.CountLooseObjects();
                     GitProcess.Result result = this.RunGitCommand((process) => process.PrunePacked(this.Context.Enlistment.GitObjectsRoot));
-                    int afterCount = this.CountLooseObjects();
+                    int afterLooseObjectsCount = this.CountLooseObjects();
+
+                    this.CreateLooseObjectsPackFile();
 
                     EventMetadata metadata = new EventMetadata();
                     metadata.Add("GitObjectsRoot", this.Context.Enlistment.GitObjectsRoot);
-                    metadata.Add("StartingCount", beforeCount);
-                    metadata.Add("EndingCount", afterCount);
-                    metadata.Add("RemovedCount", beforeCount - afterCount);
+                    metadata.Add("StartingCount", beforeLooseObjectsCount);

```

Comment:
Should we also include a message in the metadata (key `TracingConstants.MessageKey.InfoMessage`) describing this event?

Reply:
I think with your great suggestion below to change the name eventName a InfoMessage isn't needed cause it should be self describing.  Let me know if you disagree!

Reply:
That sounds good to me, thanks!